main/src/ca/uwaterloo/flix/language/phase/optimizer/OccurrenceAnalyzer.scala

```diff
@@ -540,17 +539,10 @@ object OccurrenceAnalyzer {
     /**
       * Returns a fresh [[LocalContext]].
       */
-    def mk(): LocalContext = new LocalContext(new AtomicInteger(0))
+    def mk(): LocalContext = new LocalContext()
 
   }
 
-  /**
-    * A local context, scoped for each function definition.
-    * No requirements on thread-safety since it is scoped.
-    *
-    * @param localDefs The number of declared [[Expr.LocalDef]]s in the expression.
-    *                  Must be mutable.
-    */
-  private case class LocalContext(localDefs: AtomicInteger)
+  private case class LocalContext()
```

Comment:
You can also completely remove Local Context now, right?

Reply:
Bro

![image](https://github.com/user-attachments/assets/018efa86-6647-48da-a271-6a461d9ebfc9)

Reply:
Haha sorry